### PR TITLE
Update OSIE project status from experimental to deprecated

### LINKED ISSUE #281: Deprecate OSIE
This is tracking part of the work to implement https://github.com/tinkerbell/proposals/blob/main/proposals/0025/README.md

### PR DESCRIPTION
**Pull request by tstromberg**
## Description

Fixes #281 and implements https://github.com/tinkerbell/proposals/blob/main/proposals/0025/README.md

To be merged after a final check-in at the community office hours on September 28th, 2021.

##  NOTE

OSIE has been deprecated in preference to the [Hook](https://github.com/tinkerbell/hook) project. Here is the deprecation schedule:

* September 28th, 2021: Announcement published
* November 30th, 2021: Tree closes for feature changes
* December 30th, 2021: Repository is archived (read-only)

For more details, see the [OSIE deprecation proposal](https://github.com/tinkerbell/proposals/blob/main/proposals/0025/README.md).

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,29 +1,34 @@
 # OSIE
 
 [![Build Status](https://drone.packet.net/api/badges/tinkerbell/osie/status.svg)](https://drone.packet.net/tinkerbell/osie)
-![](https://img.shields.io/badge/Stability-Experimental-red.svg)
+[![deprecated](http://badges.github.io/stability-badges/dist/deprecated.svg)](http://github.com/badges/stability-badges)
```
Comment (mmlb) [inline README.md]: Why not just change the Experimental -> Deprecated? The new badge has a completely different style.

```suggestion
![deprecated](https://img.shields.io/badge/Stability-Deprecated-red.svg)
```

**Review by ScottGarman — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -1,29 +1,38 @@
...
-This means that support is best effort (at best!) and we strongly encourage you to NOT use this in production.
+## Deprecation
+
+OSIE has been deprecated in preference to the [Hook](https://github.com/tinkerbell/hook) project. Here is the deprecation schedule:
+
+* September 28th, 2021: Announcement published
+* November 30th, 2021: Tree closes for feature changes
+* December 30th, 2021: Repository is archived (read-only)
+
+For more details, see the [OSIE deprecation proposal](https://github.com/tinkerbell/proposals/blob/main/proposals/0025/README.md).
```
Comment (splaspood) [inline README.md:18]: It feels kind of weird to be linking to a proposal, ratified or not.   What's our feeling on this?

File: README.md [hunk 3]
```diff
@@ -1,29 +1,38 @@
...
 
 ## Cloning OSIE
 
 OSIE uses git-lfs for large files that are part of the build process. If you clone this repo without git-lfs installed and set up, your builds will fail.
 
-Install git-lfs per instructions at https://git-lfs.github.com/ and make sure to run `git lfs install` afterwards to set it up in your `~/.gitconfig`.
+Install git-lfs per instructions at <https://git-lfs.github.com/> and make sure to run `git lfs install` afterwards to set it up in your `~/.gitconfig`.
```
Comment (splaspood) [inline README.md:24]: markdown link here?

File: README.md (hunk 2 above)
Comment (tstromberg) [reply README.md:18]: I think proposals are a great artifact for the contextual discussion behind large changes. I personally would suggest the idea that most repositories should have some sort of link to a proposal as to why the repository came to be, for instance.

File: README.md (hunk 3 above)
Comment (tstromberg) [reply README.md:24]: This change was from markdownlint - it seems to have done the right thing.

**Review by Raj-Dharwadkar — APPROVED** [review]
Looks good to me.

**Pull request merged**